packages/components/src/pages/InstallGuide.vue

```diff
@@ -147,41 +114,29 @@ export default {
     hasRecorded() {
       return this.numAppMaps > 0;
     },
-    hasExplored() {
-      return this.selectedProject?.appMapOpened;
-    },
-    hasInvestigatedFindings() {
-      return this.selectedProject?.investigatedFindings;
+    hasUsedNavie() {
```

Comment:
It would be good to see this exercised in the unit tests.

Reply:
I added a unit test that exercises this property [here](https://github.com/getappmap/appmap-js/pull/1585/commits/838a3e274e71ecda1c995ab1d41e30531538df46#diff-09afc86addccf37db27907dcb7fc2e304bf2f0d4908427cb7cf136ffe96cb2c8R61)